builtin/plugins/chainconfig/chainconfig.go

```diff
@@ -215,16 +215,18 @@ func (c *ChainConfig) RemoveFeature(ctx contract.Context, req *RemoveFeatureRequ
 }
 
 // ListFeatures returns info about all the currently known features.
+// Support for fetching Features Enabled via Migration is also added
 func (c *ChainConfig) ListFeatures(ctx contract.StaticContext, req *ListFeaturesRequest) (*ListFeaturesResponse, error) {
 	curValidators, err := getCurrentValidators(ctx)
 	if err != nil {
 		return nil, err
 	}
-
 	featureRange := ctx.Range([]byte(featurePrefix))
 	features := []*Feature{}
+	featureList := make(map[string]bool)
 	for _, m := range featureRange {
 		var f Feature
+		featureList[f.Name] = true
```

Comment:
This line should be below `proto.Unmarshal`

Reply:
Done.